SPECS/dosfstools/dosfstools.spec

```
@@ -1,13 +1,17 @@
 Summary:	Dos Filesystem tools
 Name:		dosfstools
-Version:	4.1
-Release:        5%{?dist}
+Version:	4.2
+Release:    1%{?dist}
 License:	GPLv3+
 URL:		http://github.com/dosfstools/dosfstools
 Group:		Filesystem Tools
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:	http://github.com/%{name}/%{name}/releases/download/v%{version}/%{name}-%{version}.tar.gz
+Source0:	https://github.com/dosfstools/dosfstools/archive/v4.2.tar.gz#/%{name}-%{version}.tar.gz
```

Let's still use `%{version}` in the URL to make updates a bit easier:

```suggestion
Source0:	https://github.com/dosfstools/dosfstools/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

---

good catch, thanks